sdk/src/main/java/com/mapbox/maps/MapInitOptions.kt

```diff
@@ -24,6 +25,7 @@ data class MapInitOptions constructor(
   var mapOptions: MapOptions = getDefaultMapOptions(context),
   var initialCameraOptions: CameraOptions? = null,
   var textureView: Boolean = false,
+  val styleUri: String? = Style.MAPBOX_STREETS,
```

Comment:
Let's also change the default value of cacheSize to `1024 * 1024 * 50` to align with iOS. cc @julianrex